src/app/task/task.component.ts

```diff
@@ -0,0 +1,40 @@
+import { TaskService } from '../task.service';
+import { Component, OnInit } from '@angular/core';
+import { Task } from './task';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-task',
+  templateUrl: './task.component.html',
+  styleUrls: ['./task.component.scss']
+})
+export class TaskComponent implements OnInit {
+  taskForm = new FormGroup({
+    name: new FormControl(''),
+    description: new FormControl(''),
+  });
+
+  tasks: Task[] = [];
+
+  constructor(public taskService: TaskService) { }
+
+  ngOnInit() { }
+
+  addTask() {
+    const name = this.taskForm.get('name').value;
+    const description = this.taskForm.get('description').value;
+    const task = new Task(1, name, description);
```

Comment:
Id is always 1. We need them to be generated randomly or to be in order 1,2,3 etc.